### DISCUSSION TITLE
How to get the addresses of the global C variables?

### DISCUSSION THREAD

**Discussion by Fabrizio-Caruso** [cc65/cc65#2143] [General]
I am looking for an option that gives me the addresses of all the global variables.

**Comment by polluks** [comment]
How about this?
```
$ cl65 -Ln global foo.c
$ grep "\._[^_]" global

**Reply by Fabrizio-Caruso** [reply]
Hi! It is not working with static variables, which are actually global with a scope only in a single file.

**Reply by polluks** [reply]
indeed

**Comment by Fabrizio-Caruso** [comment]
Thanks a lot!
